tests/ctst/steps/put-object.ts

```diff
@@ -0,0 +1,86 @@
+import {Given, Then} from '@cucumber/cucumber';
+import Zenko from 'world/Zenko';
+import {S3, Utils} from 'cli-testing';
+import {ListObjectsV2Output, ListObjectVersionsOutput, PutObjectOutput} from '@aws-sdk/client-s3';
+import {safeJsonParse} from 'common/utils';
+import assert from 'assert';
+import {saveAsFile} from './utils/utils';
+
+Given('{int} threads each uploading {int} versions of object {string} of size {int} bytes', async function (
+    this: Zenko,
+    numberOfThread: number,
+    numberOfVerionPerThread: number,
+    objectName: string,
+    sizeBytes: number,
+) {
+    const bucketName = this.getSaved<string>('bucketName');
+    this.addToSaved('objectName', objectName);
+    let processedCounter = numberOfVerionPerThread * numberOfThread;
+
+    await Promise.all(Array.from({length: numberOfThread}, async () => {
```

Comment:
```suggestion
    await Promise.all(Array.from({ length: numberOfThread }, async () => {
```

Reply:
No, I think this is also not detected by it? It was just a small nit, usually we have spaces, definitely not a problem otherwise